arbiter-core/Cargo.toml

```diff
@@ -1,14 +1,16 @@
 # Package configuration
 [package]
-name = "core"
+name = "arbiter-core"
 version = "0.3.0"
 edition = "2021"
 
 # Dependency configuration
 [dependencies]
 # For execution
+dashmap = "5.5.0"
```

Comment:
What is this used for?

Reply:
Oh we don't need anymore it is a thread safe hashmap i was playing with before i found the atomic usize